src/libs/blueprint/conduit_blueprint_mesh_examples.cpp

```diff
@@ -2270,6 +2327,61 @@ basic(const std::string &mesh_type,
 }
 
 
+//---------------------------------------------------------------------------//
+void
+basic1D(const std::string &mesh_type,
+        index_t npts_x, // number of points in x
+        Node &res)
```

Comment:
This is `basic()`, specialized for 1D mesh types and checks.  Should it should be wrapped into `basic()`?